lib/panel/dock.js

```diff
@@ -14,6 +10,11 @@ class PanelDock {
   constructor(delegate: Object) {
     this.element = document.createElement('div')
     this.subscriptions = new CompositeDisposable()
+
+    const React = require('react')
```

Comment:
Exactly my point, it's a rare occurrence so forcing the caching is better than relying on the automatic caching.